apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -118,72 +192,134 @@ export default class ExportHTMLModal extends Vue {
   // If we are exporting a profile we can remove the test/results table
   @Prop({type: String, required: true}) readonly fileType!: string;
 
+  // Generated injectable HTML for icons
+  // Icons colors should align with Vue colors used in general dashboard
```

Comment:
maybe give a path to where those colors are defined?

Reply:
Changed as requested.